FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Views/AuthPickerView.swift

```diff
@@ -28,7 +28,8 @@ extension AuthPickerView: View {
         Text(authService.authenticationFlow == .login ? "Login" : "Sign up")
         VStack { Divider() }
         EmailAuthView()
-
+        // Xcode compiler exception: Type 'any ExternalAuthProvider' cannot conform to 'ExternalAuthProvider'
```

Comment:
@peterfriese - Following your gist example, we get this Xcode compiler error now. 

I tried to remove `any ExternalAuthProvider` from the types in auth service for each provider but it isn't allowed because ExternalAuthProvider uses an associated type (ButtonType), Swift requires you to explicitly acknowledge it as an existential by using `any ExternalAuthProvider`.